totalRP3/Modules/NamePlates/NamePlates_Plater.lua

```diff
@@ -40,30 +169,31 @@ function TRP3_PlaterNamePlates:OnModuleEnable()
 		local newScriptTable = Plater.DecompressData(importString, "print");
 		local newScript = Plater.BuildScriptObjectFromIndexTable(newScriptTable, scriptType);
 
-		for i = 1, #scriptDB do
-			local scriptObject = scriptDB[i];
+		for _, scriptObject in ipairs(scriptDB) do
 			if scriptObject.Name == newScript.Name then
 				scriptAdded = scriptObject;
 			end
 		end
 	end
```

Comment:
Is it possible for there to be multiple scripts with the same name, or would it be safe for this to break after finding the first one?


```suggestion
		for _, scriptObject in ipairs(scriptDB) do
			if scriptObject.Name == newScript.Name then
				scriptAdded = scriptObject;
				break;
			end
		end
	end
```